rest-service/manager_rest/storage/storage_manager.py

```diff
@@ -399,6 +399,9 @@ def _paginate(query, pagination, get_all_results=False):
             size = config.instance.default_page_size
             offset = 0
 
+        if distinct:
+            query = query.distinct()
```

Comment:
that won't work because we're querying for models here, so this will generate just a `SELECT DISTINCT labels.key, labels.value, labels.created_by FROM labels` which means distinct on all fields, ie. *not* just distinct on keys, ie. it will then return all entries ever

instead, we need a distinct on keys, eg. "SELECT DISTINCT ON (labels.key) labels.key, labels.value, labels.created_by FROM labels" (well, we don't really need the other fields, but we'll get them anyway, because storage-manager works in terms of models)

to do that, we can do `query.distinct(models.DeploymentLabel.key)`, so the distinct param should be a column to distinct on, and not just a flag

note that "DISTINCT ON field" also requires an ORDER BY on that same field